test/conftest.py

```diff
@@ -5,17 +5,20 @@
 import discord
 import discord.ext.test as dpytest
 
+from pubobot import console, scheduler, bot as bot_, stats3, config, client
 
-@pytest_asyncio.fixture(scope="session")
+
+@pytest_asyncio.fixture(scope="session", autouse=True)
 async def bot(tmp_path_factory):
-    from pubobot import console, scheduler, bot, stats3, config, client
+    # Recreate client with the pytest-asyncio event loop
+    client.c = client.create_client()
 
     # Always use a clean database
     db = tmp_path_factory.mktemp("pubobot_tests") / "db.sqlite3"
 
     console.init(enable_input=False)
     scheduler.init()
-    bot.init()
+    bot_.init()
```

Comment:
why is this needed?

Reply:
Fixture name bot conflicts with the bot module. So it's renamed to bot_ but we can rename the fixture instead if preferred.